SumoLogic.Logging.AspNetCore/Logger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace SumoLogic.Logging.AspNetCore
+{
+    public class Logger : ILogger
+    {
+
```

Comment:
Empty row:)

Reply:
Yep. I think I need adding comments as other portions of this project as well as the markdown doc. 
Maybe Ci...
Just don't have enough bandwidth on this... ;)